approval_frame/templates/registration/password_reset_complete.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <div class='row-fluid top-buffer'>
+    <div class='col-md-12'>
+    <legend>Password reset complete</legend>
+      <p>Your password has been reset. You can now <a href="{% url 'auth_login' %}">login</a>.</p>
+    </div>
+  </div>
+{% endblock %}
```

Comment:
Sure, I realized that would make it consistent with the activation_complete.html.